website/package.json

```diff
@@ -21,6 +21,7 @@
     "clsx": "1.2.1",
     "core-js-pure": "3.29.0",
     "date-fns": "2.29.3",
+    "form-data": "^4.0.0",
```

Comment:
remove `^` and pin it. Also the other deps you added

Reply:
Oh yes!